lib/rules/forbid-elements.js

```diff
@@ -103,13 +104,11 @@ module.exports = {
           return;
         }
 
-        const argType = argument.type;
-
-        if (argType === 'Identifier' && /^[A-Z_]/.test(argument.name)) {
+        if (argument.type === 'Identifier' && /^[A-Z_]/.test(argument.name)) {
           reportIfForbidden(argument.name, argument);
-        } else if (argType === 'Literal' && /^[a-z][^.]*$/.test(argument.value)) {
+        } else if (argument.type === 'Literal' && /^[a-z][^.]*$/.test(argument.value.toString())) {
           reportIfForbidden(argument.value, argument);
-        } else if (argType === 'MemberExpression') {
+        } else if (argument.type === 'MemberExpression') {
```

Comment:
the original only has one observable lookup; this has 3. why is this an improvement?

also, `String(x)`, _never_ `x.toString()`

Reply:
> the original only has one observable lookup; this has 3. why is this an improvement?

The reason is because the following error message occurs during the execution of the type-check script.
`Property 'name' does not exist on type 'ClassExpression | ArrayExpression | ArrowFunctionExpression | AssignmentExpression | ... 23 more ... | SpreadElement'.  Property 'name' does not exist on type 'ClassExpression'.`

> also, String(x), never x.toString()

thanks! Fixed in 4ae24a2